index.html

```diff
@@ -719,6 +719,12 @@ <h3>Handling Unknown Options and Data</h3>
 Implementations MUST throw an error if an endpoint receives data, options, or option values that it
 does not understand or know how to process.
       </p>
+      <h4>Payload Sizes</h4>
+      <p>Implementations are encouraged to pay attention to the payload sizes of the Verifiable Credentials they issue.</p>
+      <p>Presentations can bundle a large volume of credentials, resulting in a higher request size than anticipated. This heightens the risk of interoperability issues.</p>
```

Comment:
`a higher request size`? 

Is this not `a higher { result | return | presentation | something-else } size`

Reply:
```suggestion
      <p>Presentations can bundle a large volume of credentials, which can result in a higher request size than anticipated by implementers and/or deployers. This raises the risk of interoperability issues.</p>
```